surveys/templates/survey_submitted_detail.html

```diff
@@ -97,16 +104,14 @@ <h3 class="card-title">Analyze</h3>
 <script src="{% static 'js/jquery-ui-1.21.1.min.js' %}" type="text/javascript"></script>
 <script>
 $(document).ready(function () {
-  $('#survey-submitted-detail').DataTable({
-    // scrollx styling taken from
-    // https://stackoverflow.com/questions/45136218/datatables-scrollx-causing-squashed-header
-    "sScrollX": "100%",
-    "sScrollXInner": "110%",
-    "fixedColumns":   {
-      "leftColumns": 1
-    }
+  var table = $('#survey-submitted-detail').DataTable({
+    buttons: [{ extend: 'csv', text: 'Download as CSV' }],
+    lengthChange: false,
   });
 
+  table.buttons().container()
+      .appendTo( '#survey-submitted-detail_wrapper .col-md-6:eq(0)' );
```

Comment:
Why do we need the additional `.col-md-6:eq(0)` selector here? Shouldn't the ID selector uniquely identify the element?

Reply:
Without `.col-md-6:eq(0)` this button would be appended to the end of the table, when instead I want it to be the first element in the row containing the search box. 

`.col-md-6:eq(0)` tells it to append the button once after an instance of a `.col-md-6` element

Reference: https://datatables.net/extensions/buttons/examples/styling/bootstrap4.html